src/common/common.css

```diff
@@ -171,6 +173,21 @@ a:active {
   margin-top: 8px;
 }
 
+@media screen and (max-width: 767px) {
```

Comment:
@rugk  I dnt think its possible that I can change element css based on its sibling element having multiline text,
but I can do a mobile first approach i.e with a min-width media query and if the button does nt fit in single line due to small size then  I will move it to next line else for all other cases it could be singleline css only.
please suggest otherwise

Reply:
Hmm I also dunno the best way. You could try to have a look on how Firefox implements it on addons.mozilla.org e.g. (look for an unsupported old extension e.g., that should have a red error message).
Or in Firefox itself (about:debugging or about:addons may be good canidates if you e.g. trigger an error).
Or in the source code… I've did search it in a similar area before, so maybe near [here](https://github.com/rugk/awesome-emoji-picker/issues/17#issuecomment-488655486) you can find something.

---

If you don't find a better way, yeah, maybe the mobile-first idea is the best. (if I understood your idea correctly)